src/streams.h

```diff
@@ -203,22 +200,19 @@ class CDataStream
     typedef vector_type::const_iterator   const_iterator;
     typedef vector_type::reverse_iterator reverse_iterator;
 
-    explicit CDataStream(int nTypeIn, int nVersionIn)
-        : nType{nTypeIn},
-          nVersion{nVersionIn} {}
-
-    explicit CDataStream(Span<const uint8_t> sp, int type, int version) : CDataStream{AsBytes(sp), type, version} {}
-    explicit CDataStream(Span<const value_type> sp, int nTypeIn, int nVersionIn)
-        : vch(sp.data(), sp.data() + sp.size()),
-          nType{nTypeIn},
-          nVersion{nVersionIn} {}
+    explicit DataStream() {}
+    explicit DataStream(Span<const uint8_t> sp) : DataStream{AsBytes(sp)} {}
+    explicit DataStream(Span<const value_type> sp) : vch(sp.data(), sp.data() + sp.size()) {}
 
     template <typename... Args>
-    CDataStream(int nTypeIn, int nVersionIn, Args&&... args)
-        : nType{nTypeIn},
-          nVersion{nVersionIn}
+    static DataStream FromMany(Args&&... args)
```

Comment:
In commit "streams: Add DataStream without ser-type and ser-version" (fa5a98f39a0b236be0ce468d71169a560f2dc489)

What's going on with this `FromMany` constructor? It seem unused. Would be good to simplify the commit and drop it

Reply:
Dropped

Reply:
Also removed the constructor in https://github.com/bitcoin/bitcoin/pull/26992